src/jquery.i18n.fallbacks.js

```diff
@@ -1,4 +1,4 @@
-/*!
+/* !
```

Comment:
By "you" i meant wikimedia team. 

> because otherwise it would complain about @license tags being unknown tags. If it is no longer deemed necessary, then ! should be removed.

Not sure what do you mean by that, i would advise for you to discuss this among yourself  and decide what you wanna do